### DISCUSSION TITLE
I NEED HELP!

### DISCUSSION THREAD

**Discussion by scottbgreene** [Jyers/Marlin#1158] [Q&A] [answered]
Hello,

I'm new to all of this but I am trying to learn and need some help. I have an ender 3 max. I also have the screen and motherboard for an Ender 3v2 I had. I put the screen and motherboard on the Max and installed the Jyers firmware without a bl touch. Well it works fine except that it doesn't recognize the full size of the bed (I changed the bed settings in cura) and I need to know how to change those settings so it realizes it's on a bigger bed. I really hope someone can help me with this. I am sure it is an easy fix. I just don't know how to do it or where to start.

Cheers!

**Comment by Cal-LF** [comment, marked as answer]
Jyres FW does not have a bed size setting in the menu. In order for you to change the bed size, you will need to compile firmware with one of the templates in the Configuration Files. Default-NoProbe or ManualMesh-3x3 configs.
Another option is to become a member of Jyres patreon. https://www.patreon.com/jyers compile service for $5 a month.
 Compile your custom setup and get it the next hour.

**Reply by scottbgreene** [reply]
Thank you for the response. I am lost when it comes to doing that. I know I need to learn it. I have no problem with paying that. I know I paid $15 for a year - Twice by mistake - for a similar service somewhere else but it was not for this. I really like the Jyers options it gives. I am going to go to the patreon and sign up for that. I am sure there will be instructions on how to do it. Thanks again for the help!
